projects/gstreamer/gst-discoverer.c

```diff
@@ -0,0 +1,505 @@
+/* GStreamer
```

Comment:
Yea, we encourage to have the fuzz target in the project's repository, not here. 
https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md

Reply:
Also check the license. 
We don't want files with such license in this repository.